templates/terraform/resource.erb

```diff
@@ -232,12 +266,21 @@ func resource<%= resource_name -%>Update(d *schema.ResourceData, meta interface{
 	<% end -%>
 	}
 
+  <% if object.custom_code.encoder or object.custom_code.update_encoder -%>
```

Comment:
Use `||` instead:
- https://github.com/bbatsov/ruby-style-guide#no-and-or-or
- https://stackoverflow.com/questions/2083112/difference-between-or-and-in-ruby

Reply:
Ruby ruby ruby ruby.  :(